versions.tf

```diff
@@ -3,12 +3,12 @@
 # ----------------------------------------------------------------------------------------------------------------------
 
 terraform {
-  required_version = "~> 1.0"
+  required_version = ">= 0.14.7, < 2.0"
```

Comment:
Please revert this and keep the version as previously defined. We do not add support for previous major versions anymore.